components/org.wso2.carbon.siddhi.editor.core/src/main/java/org/wso2/carbon/siddhi/editor/core/internal/ServiceComponent.java

```diff
@@ -412,6 +412,24 @@ public Response read(String path) {
     }
 
     @POST
+    @Path("/workspace/read/sample")
```

Comment:
@suhothayan we have already method path for read as @Path("/workspace/read") and this @Path("/workspace/read/sample") will get the request and append the carbon_home to path and call above same read method. since we have a path called @Path("/workspace/read") and here we are going to read samples, hence i put it as @Path("/workspace/read/sample") . WDYT ?